mkosi/kmod.py

```diff
@@ -150,12 +150,9 @@ def resolve_module_dependencies(
     return set(nametofile[m] for m in mods if m in nametofile), set(firmware)
 
 
-def parents_relative_to(path: Path, other: Path) -> Iterator[Path]:
-    for p in path.parents:
-        if p == other or not p.is_relative_to(other):
-            return
-
-        yield p
+def parents_until(path: Path, until: Path) -> list[Path]:
```

Comment:
I think that name would be a bit clearer
```suggestion
def parents_below(path: Path, until: Path) -> list[Path]:
```
since—to me—until  doesn't imply whether the bound is part or not, while below makes it clear that all of them should be children.